WebHostLib/templates/options.yaml

```diff
@@ -46,6 +46,9 @@ requires:
     {%- for suboption_option_id, sub_option_name in option.name_lookup.items() %}
     {{ sub_option_name }}: {% if suboption_option_id == option.default %}50{% else %}0{% endif %}
     {%- endfor -%}
+    {% if option.default == "random" %}
```

Comment:
Does this not leave unwanted newlines behind? Normally one has to use {%- and/or -%} to prevent printing newlines for control characters.

Reply:
So that specific line did not add an extra newline, but technically there is an extra newline after the "random" entry. I just pushed a slight tweak that removes that. See below for example output:

```yaml
Rogue Legacy:
  starting_gender: # 
    #    Determines the gender of your initial 'Sir Lee' character.
    #    
    sir: 0
    lady: 0
    random: 50
  starting_class: # 
    #    Determines the starting class of your initial 'Sir Lee' character.
    #    
    knight: 50
    mage: 0
    barbarian: 0
    knave: 0
    shinobi: 0
    miner: 0
    spellthief: 0
    lich: 0
```